politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -593,6 +676,27 @@ func validateCockroachParams() error {
 	return nil
 }
 
+func validateMySQLParams() error {
+	// Validate host.
+	_, err := url.Parse(*mysqlhost)
+	if err != nil {
+		return fmt.Errorf("parse host '%v': %v", *mysqlhost, err)
+	}
+
+	// Validate password.
+	if *password == "" {
+		return fmt.Errorf("MySQL password missing; use -password flag to" +
```

Comment:
For what user? I don't know if this is asking for the root password or politeiawww password.